ChangeLog.d/fix_psa_crypto_cipher_h_include

```diff
@@ -0,0 +1,2 @@
+Bugfix
```

Comment:
This file should have a .txt extension, or the script that processes these files will miss it, causing extra work in the release process :(

Reply:
@tom-cosgrove-arm It's my first PR so I just copy-pasted the file from another PR. I have now modified the file extension... please review and let me know what is missing! Thanks, Martin!

Reply:
@mman Thanks for fixing that! Unfortunately, the CI requires every commit to have a DCO (`Signed-off-by:`) line. Since this PR is so small, I would suggest that you just rebase and squash all 3 commits, and then push a single commit with the DCO line.

(In general we don't like people modifying commits after they've been looked at on a PR, as it makes reviewing harder, but this is so small rebasing and squashing will make things easier)

Reply:
@tom-cosgrove-arm Should be addressed now...